libtenzir/builtins/contexts/bloom_filter.cpp

```diff
@@ -163,18 +167,17 @@ class bloom_filter_context final : public virtual context {
                          .make_query = std::move(query_f)};
   }
 
-  auto update(chunk_ptr, context::parameter_map)
-    -> caf::expected<update_result> override {
-    // TODO: We're getting chunks piece-meal currently and therefore cannot
-    // determine the boundary of the stream. In theory, we should accumulate the
-    // chunks until the upstream pipeline is complete. What we need in the
-    // context API is a signal that we're done.
-    return ec::unimplemented;
+  auto make_query() -> caf::expected<make_query_type> override {
+    return caf::make_error(ec::unimplemented,
+                           "bloom filter doesn't support retroactive lookups "
+                           "after insertion");
```

Comment:
```suggestion
    return {};
```

This error gets swallowed by the post-load query expression emittance loop. It will never be displayed.

Contexts that do not support retroactive lookups should return an empty function.

If anything, this error should appear when you want to _only_ create a retrospective lookup for a context that doesn't support them, but that's not in the scope of this PR.